lib/ruby_indexer/lib/ruby_indexer/entry.rb

```diff
@@ -203,14 +211,28 @@ def list_params(parameters_node)
           end
         end
 
+        rest_name = parameters_node.rest&.name
+        parameters << RestParameter.new(name: rest_name) if rest_name
+
+        keyword_rest_name = parameter_name(parameters_node.keyword_rest)
+        parameters << KeywordRestParameter.new(name: keyword_rest_name) if keyword_rest_name
+
+        parameters_node.posts.each do |post|
+          name = parameter_name(post)
+          next unless name
+
+          parameters << RequiredParameter.new(name: name)
```

Comment:
We don't, we store `(y, z)` as a single required parameter exactly for that reason. Especially because you can just pass a variable, which would be a single parameter:

```ruby
def foo((y, z))
end

a = [1, 2]

foo(a) # good
```